### ISSUE TITLE
INVALID_ARGUMENT : Unexpected input data type.Actual: (class onnxruntime::PrimitiveDataType<double>) , expected: (class onnxruntime::PrimitiveDataType<float>)

### ISSUE THREAD

**Issue by Nathanielll** [onnx/keras-onnx#323]
my code：
———————————————————————————————————————————
import numpy
from sklearn.datasets import load_boston
from keras import models
import keras
import onnx

import onnxmltools
import onnxruntime

x = load_boston().data.astype(numpy.float32)
y = load_boston().target.astype(numpy.float32)

model = keras.Sequential()
model.add(keras.layers.Dense(8))
model.add(keras.layers.Dense(1))

model.compile(keras.optimizers.Adam(0.001, ), loss='mse')

model.fit(x, y, epochs=10)
model.save('model.h5')

model = models.load_model('model.h5')

onnx_net = keras2onnx.convert_keras(model)

onnxmltools.utils.save_model(onnx_net, 'model.onnx')

temp_model_file = 'model.onnx'
onnx_net = onnx.load_model(temp_model_file)
content = onnx_net.SerializeToString()

sess = onnxruntime.InferenceSession(temp_model_file)

for i in sess.get_inputs():
    print(i.name)
    print(i.shape)
sess.get_outputs()
for i in sess.get_outputs():
    print(i.name)
    print(i.shape)

x = numpy.random.random((1, 13)).astype(numpy.float64)
x = x if isinstance(x, list) else [x]
feed = dict([(input.name, x[n]) for n, input in enumerate(sess.get_inputs())])
print(sess.run(None,
               feed))

console:
——————————————————————————————————————————
    return self._sess.run(output_names, input_feed, run_options)
onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Unexpected input data type. Actual: (class onnxruntime::PrimitiveDataType<double>) , expected: (class onnxruntime::PrimitiveDataType<float>)

**Comment by jiafatom** [comment]
See the error message, it means you have the wrong input type, try `x = numpy.random.random((1, 13)).astype(numpy.float32)` not `float64`.

**Comment by Nathanielll** [comment]
i tried **x = numpy.random.random((1, 13)).astype(numpy.float32)**  before, but it didn't work.  

i have solved it，thanks for ur  help.
